app/routes/monthlychallenges/july-2022.jsx

```diff
@@ -0,0 +1,176 @@
+import { json } from '@remix-run/node';
+import { createMetaData } from '~/util/createMetaData.server';
+
+export const handle = {
+	listTitle: 'July, 2022: Demo in public!',
+	meta: {
+		title: 'Monthly Theme & Challenge for July, 2022: Build in Public',
+		description:
+			"July challenge -> Let's build in public, starting with Daily Standup! During this month, let's learn to communicate what we're working on, show our development, be confident and proud of any progress we've made, and demo what we've done!",
+	},
+	date: '2022-07-01',
+	hero: {
+		heroHeader: '',
+	},
+};
+
+export async function loader() {
+	const { title, description } = handle.meta;
+	return json({
+		meta: createMetaData({ title, description }),
+	});
+}
+
+export function meta({ data: { meta } = {} } = {}) {
+	return meta;
+}
+
+export default function Challenge() {
+	return (
+		<>
+			<div className="alert alert-success">
+				This monthly challenge is complete. Congratulations! Please join us for
+				the <a href="/monthlychallenges/">next challenge</a>!
+			</div>
+
+			<h1>
+				<small>Monthly Challenge for July, 2022:</small> Let's build in public,
+				starting with Daily Standup!
+			</h1>
+
+			<p className="lead">
+				During this month, let's learn to communicate what we're working on,
+				show our development, and be confident and proud of any progress made.
+			</p>
+
+			<hr />
+
+			<h2>Theme</h2>
+			<p>Demo in public!</p>
+
+			<h2>Goals</h2>
+			<p>To end the month with:</p>
+			<ul>
+				<li>A habit of talking about the things we're working on.</li>
+				<li>A plan for continuing progress.</li>
+				<li>A demo for the VC community.</li>
+			</ul>
+
+			<h3>Who can participate?</h3>
+			<p>
+				Virtual Coffee's goal is to support all developers, no matter where they
+				are in their coding journey. This challenge is opened to non-coding
+				projects as well. So if you want to talk about the projects you're doing
+				in other areas of your life, we want to hear about it. We encourage all
+				members to participate. This challenge can be done alone or in a group.
+			</p>
+
+			<h3>How it works</h3>
```

Comment:
I know this was in the content, but I think we can remove the <h3> with "How it works".